Swerve/SwerveModules/SwerveModuleTalon.java

```diff
@@ -83,4 +83,9 @@ public void runSysIdRoutine(double voltage) {
     protected Voltage getDriveVoltage() {
         return driveMotor.getMotorVoltage().getValue();
     }
+    
+    @Override
```

Comment:
you are overriding this twice making the base class implementation useless, consider making another abstract method and calling that method in the updateInputs method in the base class to give the child class the opportunity to populate certain inputs.

Reply:
Git put this in a odd spot through the commits

Reply:
it fixed now